frontend/src/Pages/AllUser/Login/ChangePassword.jsx

```
@@ -0,0 +1,100 @@
+  return (
+    <div className="flex items-center justify-center min-h-screen px-4 text-white bg-gray-900">
+      <div className="w-full max-w-md p-8 bg-gray-800 shadow-lg rounded-2xl">
+        <h2 className="mb-6 text-2xl font-bold text-center">Change Password</h2>
+
+        {message && <div className="p-3 mb-4 text-green-700 bg-green-100 rounded">{message}</div>}
+        {error && <div className="p-3 mb-4 text-red-700 bg-red-100 rounded">{error}</div>}
+
+        <form onSubmit={handleSubmit} className="space-y-4">
+          <div>
+            <label className="block mb-1 text-sm font-medium text-gray-300">Current Password</label>
+            <input
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)